index.html

```diff
@@ -0,0 +1,62 @@
+<!--  Primary Requirements
+
+Add pages to your static site. The requirement is a minimum of four pages. These pages should be:
+index.html
+portfolio.html: information about the projects you've completed at Ada (or elsewhere) with links (to github repo), descriptions, images, etc.
+code-journal.html or hobby-blog.html: article or blog style posts about either your journey/observations about programming or a hobby you enjoy.
+about.html: some information about you, your interests, background or similar. Whatever you're comfortable sharing.
+The site should follow best practices including:
+All markup should be semantic, with consideration of hierarchy and accessibility.
+CSS should be concise and well formatted
+Images and stylesheets should be kept in their own folders, called images and stylesheets, respectively.
+
+ -->
+
+ <!DOCTYPE html>
+ <html>
+   <head>
+     <meta charset="utf-8">
+     <title>Angela Wilson Homepage</title>
+     <link rel="stylesheet" href="styles/style.css">
+   </head>
+   <body>
+     <header>
+       <nav>
+         <ul>
+           <li>
+             <a href="index.html">Home</a>
+           </li>
+           <li>
+             <a href="Portfolio.html">Portfolio</a>
+           </li>
+           <li>
+             <a href="Code-Journal.html">Code Jounal</a>
+           </li>
+           <li>
+             <a href="About.html">About</a>
+           </li>
+         </ul>
+       </nav>
+     </header>
+
+     <video autoplay loop>
+       <source src="video/632332824.mp4" type="video/mp4">
+     </video>
+
+
+      <article class="text-align-center">
+        <h1>
+          Student Full-Stack Developer at Ada Developers Academy
+        </h1>
+        <h2>
+          Let's transform tech by effecting one mind at a time
+        </h2>
+     </article>
+
+     <!-- Footer -->
+     <footer>
```

Comment:
We should have this `footer` section on every page, similarly to the `header` tag. This will help maintain a consistent styling and layout throughout the site.